README.md

```diff
@@ -182,11 +182,69 @@ In addition to basic REST requests, the Brainstem.js model `fetch` method suppor
 ```javascript
 Post = require('./models/post');
 
-new Post({ id: 1 }).fetch({ include: ['user', 'comments'] })
+new Post({ id: 1 })
+  .fetch({ include: ['user', 'comments'] })
   .done(/* handle result */)
   .fail(/* handle error */);
 ```
 
+Brainstem also supports requesting deeply nested associations. Under the hood, Brainstem makes recursive requests for each layer of associations. There are three ways to make this kind of request, using JSON Objects, Backbone Collections or BrainstemParams. Using Backbone Collections or BrainstemParams enables expressing additional options for the nested includes / associations.
+
+##### JSON Object Example
+
+```javascript
+Post = require('./models/post');
+
+new Post({ id: 1 })
+  .fetch({ 
+    include: [
+      'user', 
+      { comments: ['replies', 'ratings'] }
+    ]
+  })
+  .done(/* handle result */)
+  .fail(/* handle error */);
+```
+
+##### Backbone Collection Example
+
+```javascript
+Post = require('./models/post');
+Comments = require('./collections/comments')
+
+var comments = new Comments(null, {
+  include: ['replies', 'ratings'],
+  filters: { only_by_user_id: 1 }
+});
+
+new Post({ id: 1 })
+  .fetch({ include: ['user', { comments: comments }] })
+  .done(/* handle result */)
+  .fail(/* handle error */);
+```
+
+In this example we are able to specify a filter for the comments association. With the filter, we can request comments for the post filtered by user_id 1.
+Here, `comments` needs to be the name of the association that post has for comments.
+
+##### BrainstemParams Example
```

Comment:
Could we add more of an explanation about what BrainstemParams is and why we need it? It seems like the answer is "use BrainstemParams if you want to have additional options but don't want to use a collection" but that's not totally clear from the text here - reading this, I'm not sure if I should use BrainstemParams cause I don't know if it appears somewhere else or includes extra behavior I don't want.

Reply:
👍